tools/check/check.go

```diff
@@ -16,50 +16,101 @@
 package main
 
 import (
+	"encoding/hex"
 	"errors"
 	"flag"
 	"fmt"
 	"io"
 	"os"
+	"strconv"
+	"strings"
 	"time"
 
+	"github.com/google/go-sev-guest/tools/lib/cmdline"
 	"github.com/google/go-tdx-guest/abi"
+	checkpb "github.com/google/go-tdx-guest/proto/check"
+	pb "github.com/google/go-tdx-guest/proto/tdx"
+	testcases "github.com/google/go-tdx-guest/testing"
+	"github.com/google/go-tdx-guest/validate"
 	"github.com/google/go-tdx-guest/verify"
 	"github.com/google/go-tdx-guest/verify/trust"
 	"github.com/google/logger"
+	"go.uber.org/multierr"
 	"google.golang.org/protobuf/encoding/prototext"
 	"google.golang.org/protobuf/proto"
-
-	pb "github.com/google/go-tdx-guest/proto/tdx"
-	testcases "github.com/google/go-tdx-guest/testing"
 )
 
 const (
 	defaultCheckCrl      = false
 	defaultGetCollateral = false
 	defaultTimeout       = 2 * time.Minute
 	defaultMaxRetryDelay = 30 * time.Second
+	defaultMinQeSvn      = 0
+	defaultMinPceSvn     = 0
 
 	// Exit code 1 - tool usage error.
 	exitTool = 1
 	// Exit code 2 - quote verification error.
 	exitVerify = 2
 	// Exit code 3 - network error while downloading collateral.
 	exitNetwork = 3
+	// Exit code 4 - the quote did not validate according to policy.
+	exitPolicy = 4
```

Comment:
Would it make sense to have it more explicit i.e. exitPolicyMismatch or something like this?

Reply:
I tried to keep the naming convention similar to other exit codes. If required, I can change this.